src/clj/doit/util.clj

```diff
@@ -5,3 +5,7 @@
 
 (defn not-expired? [exp]
   (< (current-unix-time) exp))
+
+(defn wrap-response [data status]
```

Comment:
Shouldn't this go into your middlewares?

Reply:
It is not a middleware per se, because it is higher level function is not called on the "top" of a request handler. It is merely a convenience so that I don't have to write the map explicitly, though not sure if I should even have this function.

Reply:
If you'd like to keep it then probably just call it `response` and not `wrap-response` since wrap is typically used for middlewares / functions that _wrap_ over other functions.

Reply:
Makes sense, will rename it to `response*`, apparently `*` at the end is a convention for helper function.